backend/README.md

```diff
@@ -45,9 +45,9 @@ Refer to the CLI documentation [here](https://docs.microsoft.com/en-us/ef/core/m
 
 # Database Migration Management
 
-The DB is setup and configured through Entity Framework Code-First processes.
+The DB is setup and configured through Entity Framework Code-First processes. All dotnet ef commands must be run from the /dal directory.
 
-To use the Entity Framework CLI you will need to install the **.NET SDK version**, **dotnet-ef tool** and add a `connectionstrings.json` configuration file.
+To use the Entity Framework CLI you will need to install the **.NET SDK version**, **dotnet-ef tool** and add a `connectionstrings.json` configuration file to /dal.
```

Comment:
See the above comment related to the choice of using the `connectionstrings.json` or `.env` file.